src/peer_connection/mod.rs

```diff
@@ -1262,6 +1257,17 @@ impl RTCPeerConnection {
                         continue;
                     }
 
+                    let mid_value = match get_mid_value(media) {
+                        Some(mid) => {
```

Comment:
```suggestion
                        Some(mid) if !mid.empty() => { mid }
                        _ => continue
```

Reply:
I feel like there was a reason I didn't do this, but I can't remember what it was anymore :D